server/db_models/README.md

```diff
@@ -0,0 +1,21 @@
+## Database setup
+
+1. The postgreSQL server is running, and the database you want to use is ready.
+2. Environment variable DB_CONNECTION_STRING is set to the connect string for your postgres server. See default string in config.py for an example.
+3. Environment variable FLASK_APP is set to access app.py. 
+    - Other environment variables are set appropriately.
+4. (optional) Do a check with previous basic routes: 
+    - start the app (`flask run`)
+    - localhost:5000/v1/welcome
+    - `curl -X POST -H 'Content-Type: application/json' -d '{"teamName": "sally"}' http://localhost:5000/v1/ping`
+    - stop the app
+5. Ensure FLASK-SQLAlchemy, FLASK-Migrate and psycopg2 are installed in your virtual environment.
```

Comment:
It's probably better to add these packages to the Pipfile and just have this step say to run "pipenv install".

Reply:
These are setup issues that can be addressed when making our mid-term "release", together with other packages. Please assign yourself to this task.

Reply:
I'd recommend addressing them now, before we start building things that require different versions.